### PR TITLE
pyzbar wrapper of zbar lib

### PR DESCRIPTION
**Pull request by mrakitin**
Should be merged along with #311.

Both zbar and pyzbar packages were successfully built and tested locally on https://hub.docker.com/r/nsls2/debian-with-miniconda/.

### CODE REVIEW

File: recipes-tag/pyzbar/run_test.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+import pyzbar
+import pyzbar.pyzbar
+from pyzbar.pyzbar import decode
```
Comment (jrmlhermitte) [inline recipes-tag/pyzbar/run_test.py]: I'm not familiar. Will this get automatically tested? Also why not just put it in the test import key of the meta.yml?

Comment (mrakitin) [reply recipes-tag/pyzbar/run_test.py]: Yes, it's automatically tested. The reason is that you cannot directly import it with `import pyzbar.pyzbar.decode`.

**Comment by jrmlhermitte** [comment]
Looks good to me. merged, thanks @mrakitin !

**Comment by mrakitin** [comment]
Thanks @jrmlhermitte!

File: recipes-tag/pyzbar/run_test.py (hunk 1 above)
Comment (mrakitin) [reply recipes-tag/pyzbar/run_test.py]: I haven't found an example how you can include `from pyzbar.pyzbar import decode` into the `meta.yml` directly. If you encountered something like this, I'll be happy to get rid of the `run_test.py` file.

Comment (mrakitin) [reply recipes-tag/pyzbar/run_test.py]: Some docs about test files: https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html?highlight=run_test#test-section

**Pull request merged**